x-pack/legacy/plugins/ml/server/routes/job_service.ts

```diff
@@ -0,0 +1,609 @@
+        const { datafeedIds } = request.body;
+        const resp = await stopDatafeeds(datafeedIds);
+
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/delete_jobs
+   * @apiName DeleteJobs
+   * @apiDescription Deletes an existing anomaly detection job
+   */
+  router.post(
+    {
+      path: '/api/ml/jobs/delete_jobs',
+      validate: {
+        body: schema.object(jobIdsSchema),
+      },
+    },
+    licensePreRoutingFactory(xpackMainPlugin, async (context, request, response) => {
+      try {
+        const { deleteJobs } = jobServiceProvider(context.ml!.mlClient.callAsCurrentUser);
+        const { jobIds } = request.body;
+        const resp = await deleteJobs(jobIds);
+
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/close_jobs
+   * @apiName CloseJobs
+   * @apiDescription Closes one or more anomaly detection jobs
+   */
+  router.post(
+    {
+      path: '/api/ml/jobs/close_jobs',
+      validate: {
+        body: schema.object(jobIdsSchema),
+      },
+    },
+    licensePreRoutingFactory(xpackMainPlugin, async (context, request, response) => {
+      try {
+        const { closeJobs } = jobServiceProvider(context.ml!.mlClient.callAsCurrentUser);
+        const { jobIds } = request.body;
+        const resp = await closeJobs(jobIds);
+
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/jobs_summary
+   * @apiName JobsSummary
+   * @apiDescription Creates a summary jobs list. Jobs include job stats, datafeed stats, and calendars.
+   */
+  router.post(
+    {
+      path: '/api/ml/jobs/jobs_summary',
+      validate: {
+        body: schema.object(jobIdsSchema),
+      },
+    },
+    licensePreRoutingFactory(xpackMainPlugin, async (context, request, response) => {
+      try {
+        const { jobsSummary } = jobServiceProvider(context.ml!.mlClient.callAsCurrentUser);
+        const { jobIds } = request.body;
+        const resp = await jobsSummary(jobIds);
+
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/jobs_with_timerange
+   * @apiName JobsWithTimerange
+   * @apiDescription Creates a list of jobs with data about the job's timerange
+   */
+  router.post(
+    {
+      path: '/api/ml/jobs/jobs_with_timerange',
```

Comment:
This should have been spotted before, but might be a good time to switch this path to `/api/ml/jobs/jobs_with_time_range` as 'timerange' is not a word AFAIK. Ideally the client-side function `jobsWithTimerange` in `public/application/services/ml_api_service/jobs.js` should be edited too, but that might be too much code churn for this PR.

Reply:
Updated to `jobs_with_time_range` (except client side) in 9d1014383ec15daef0ec6402379d54d412360d15